_posts/2022-05-23-gloria-package-relay.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Gloria Zhao & Package Relay"
+tags:
+  - p2p
+  - package-relay
+  - mempool
+  - knowledge-management
+---
+
+Gloria Zhao talks with us about her package relay proposal.
+
+<iframe src="TODO" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>
+
+[anchor.fm](TODO)
+
+- What's package relay? (1:04)
+- Why do people care about package relay? (3:12)
+- What are these "contracting protocols" package relay matters for? (5:03)
+    - Pinning attacks (6:28)
+- Why do you work on package relay? (6:55)
+- What's special about the mempool? (10:18)
+- How do you approach the security considerations? (12:07)
+- Synthesizing information for the ones coming after you (15:27)
+- What's next for package relay? (17:50)
+- Bridging protocol development with L2 (20:55)
+
+Additional resources:
+- [Brink Podcast: Ep1 Mempool Policy](https://brink.dev/podcast/1-mempool-policy/)
+- [Censorship and DoS Attacks: An intro to Mempool Policy](https://vimeo.com/704956163)
+- [Transaction Relay Policy for L2 Developers](https://www.youtube.com/watch?v=fbWSQvJjKFs)
+- [Package Mempool Examples](https://raw.githubusercontent.com/glozow/bitcoin-notes/master/images/package_mempool_examples.jpg)
```

Comment:
I think this can be deleted since it's also in the mempool garden.

Would https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-September/019464.html or https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-May/020493.html be relevant?

Reply:
Thanks, removed the extraneous link and added the two mailing list posts.

Reply:
(Subsequently realized it's not actually in the mempool garden, sorry about that, but still think this is better) thanks!